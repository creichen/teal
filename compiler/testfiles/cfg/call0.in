fun g() = {
  return 1;
}
fun f() = {
  return g();
}

