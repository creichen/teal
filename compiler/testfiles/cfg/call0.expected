Entry#0 @ 1 :
  -> IntConstant#0
ReturnStmt#0 @ 2 :
  -> Exit#0
IntConstant#0 @ 2 :
  -> ReturnStmt#0
Exit#0 @ 3 :
Entry#1 @ 4 :
  -> CallExpr#0
ReturnStmt#1 @ 5 :
  -> Exit#1
CallExpr#0 @ 5 :
  -> ReturnStmt#1
Exit#1 @ 6 :
